res/css/views/elements/_Field.pcss

```diff
@@ -53,6 +53,8 @@ limitations under the License.
 .mx_Field select {
     -moz-appearance: none;
     -webkit-appearance: none;
+    width: 0;
```

Comment:
I think all you need to do is change the grid-template [here](https://github.com/matrix-org/matrix-react-sdk/blob/germain-gg/fix-right-panel-member/res/css/views/settings/tabs/_SettingsSection.pcss#L31) to:
```css
    grid-template-columns: minmax(0, 1fr);
```

By default, both grid and flex children have `min-width` set to auto which **does not** resolve to zero. Instead of setting min-width on the children, we can just change it in the parent grid.

You can set `text-overflow:ellipsis;` on the select element as well.

Reply:
I'll make the changes and thank you for the help.